packages/core/src/components/carousel/carousel.scss

```diff
@@ -21,3 +21,42 @@
     border-color: var(--color-brand-secondary-dark-1);
   }
 }
+
+::part(container) {
+  padding-bottom: var(--spacing-giant);
+}
+
+::part(thumb) {
+  align-items: center;
+  background: var(--color-neutral-white);
+  border: 1px solid var(--color-neutral-light-4);
+  border-radius: var(--border-radius-medium);
+  cursor: pointer;
+  display: inline-flex;
+  height: var(--spacing-giant);
+  justify-content: center;
+  opacity: 1;
+  padding: var(--spacing-xxsmall);
+  width: var(--spacing-giant);
+}
+
+::part(thumb-youtube-icon) {
+  color: red;
+  font-size: 30pt;
```

Comment:
(block) why `pt`? we need to use `px` or `em`

Reply:
i changed to the variable, i forgot to change it since it was one of the first things i did in this feature :sweat: 

[5c47efd](https://github.com/juntossomosmais/atomium/pull/290/commits/5c47efd5dd7f7ec230884f1a402c8fce6453d145)